alacritty_terminal/src/term/mod.rs

```diff
@@ -308,7 +302,61 @@ pub struct Term<T> {
     damage: TermDamageState,
 
     /// Config directly for the terminal.
-    config: Terminal,
+    options: TermOptions,
+}
+
+/// Configuration options for the [`Term`].
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct TermOptions {
```

Comment:
Maybe just name it Options/Config? Term should be clear due to module path.

Reply:
I guess, it's just we name everything with `Term` here, like `TermMode`.